package.json

```diff
@@ -4,6 +4,11 @@
   "workspaces": {
     "packages": [
       "kebab",
+      "kebab/examples/bot",
```

Comment:
nooooo, don't add them to workspaces. just make them use yarn outside of workspaces

Reply:
it wouldn't let me because they were not in a workspace. I'll see what I can do though

Reply:
done :)